Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "perf-zoo"
+version = "0.1.0"
+authors = ["Yao Yue <thinkingfish@gmail.com>"]
+edition = "2018"
+
+[dependencies]
```

Comment:
Consider using the time crate for timing with precise_time_ns()

```
[dependencies]
time = "0.1.42"
```

I thought it was interesting to change the output slightly (see below)